src/js/components/RegisterPage/SignUp.js

```diff
@@ -155,7 +155,7 @@ class SignUp extends Component {
 
                                 <div className="tab-content">
                                     <div className="tab-pane container active" id="home">Some interesting information, when I say interesting I mean really interesting</div>
-                                    <div className="tab-pane container" id="menu1">Some interesting information, when I say interesting I mean really interesting</div>
+                                    <div className="tab-pane container" id="menu1">Thought I found a way , Isn't it lovely say interesting I mean really interesting</div>
```

Comment:
i will prefer you  writing text inside a span or paragraph of some sort not directly inside the div element

Reply:
Thanks it is now contained in paragraphs